internal/provider/schemas/team.go

```diff
@@ -67,3 +74,81 @@ func TeamDataSourceSchemaAttributes() map[string]datasourceSchema.Attribute {
 		},
 	}
 }
+
+func TeamResourceSchemaAttributes() map[string]resourceSchema.Attribute {
+	return map[string]resourceSchema.Attribute{
+		"id": resourceSchema.StringAttribute{
+			MarkdownDescription: "Team identifier",
+			Computed:            true,
+			PlanModifiers: []planmodifier.String{
+				stringplanmodifier.UseStateForUnknown(),
+			},
+		},
+		"name": resourceSchema.StringAttribute{
+			MarkdownDescription: "Team name",
+			Required:            true,
+		},
+		"description": resourceSchema.StringAttribute{
+			MarkdownDescription: "Team description",
+			Optional:            true,
+		},
+		"member_ids": resourceSchema.SetAttribute{
+			ElementType:         types.StringType,
+			MarkdownDescription: "The IDs of the users to add to the Team",
+			Optional:            true,
+			Validators: []validator.Set{
+				setvalidator.SizeAtLeast(1),
```

Comment:
does it have to be 1, can we have empty team?

Reply:
yes we can have empty team - will change